peripydic/materials/elastic.py

```diff
@@ -73,8 +73,6 @@ def compute_dilatation_slice(self, deck, data_solver, y, start, end):
                     self.dilatation[i] += (1. / self.Weighted_Volume[i]) * functions.w(data_solver, X, deck.influence_function) * linalgebra.norm(X) * self.e[i,n] * self.Volume_Correction[i,n] * deck.geometry.volumes[p]
 
                 if deck.dim == 2:
-                    if self.Weighted_Volume[i] == 0:
```

Comment:
@diehlpk What was this for ? Do we not need it at all anymore ?

Reply:
It was in the previous commit but removed in the current one. Was some relict from debugguing.

Reply:
Alright.